src/main/webapp/style.css

```diff
@@ -17,5 +17,12 @@ button.btn {
 
 button.btn:hover {
   box-shadow: 0 12px 16px 0 rgba(0,0,0,0.24),0 17px 50px 0 rgba(0,0,0,0.19);
+}
+
+.card-holder {
+  transition: .05s;
+}
 
+.card-holder:hover {
+  box-shadow: 0 12px 16px 0 rgba(0,0,0,0.24),0 17px 50px 0 rgba(0,0,0,0.19);
 }
```

Comment:
Thanks for making a generalized version! Will add to mine in my next PR.

Reply:
Yeah! I didn't notice that! Will use!